tests/unit/nupic/docs/examples_test.py

```diff
@@ -24,50 +24,145 @@
+    for i in range(len(ExamplesTest.oneStepPredictions["opf"])):
+      self.assertEquals(ExamplesTest.oneStepPredictions["opf"][i],
+                        ExamplesTest.oneStepPredictions["algo"][i])
+
+
+  @unittest.skip
+  def testOneStepPredictionsOpfVsNetwork(self):
+    """Make sure one-step predictions are the same for OPF and Network API."""
+    for i in range(len(ExamplesTest.oneStepPredictions["opf"])):
+      self.assertEquals(ExamplesTest.oneStepPredictions["opf"][i],
+                        ExamplesTest.oneStepPredictions["network"][i])
+
+
+  @unittest.skip
+  def testOneStepPredictionsAlgoVsNetwork(self):
+    """Make sure one-step predictions are the same for Algo and Network API."""
+    for i in range(len(ExamplesTest.oneStepPredictions["algo"])):
+      self.assertEquals(ExamplesTest.oneStepPredictions["algo"][i],
+                        ExamplesTest.oneStepPredictions["network"][i])
+
+
+  @unittest.skip
+  def testFiveStepPredictionsOpfVsNetwork(self):
+    """Make sure five-step predictions are the same for OPF and Network API."""
+    for i in range(len(ExamplesTest.fiveStepPredictions["opf"])):
+      self.assertEquals(ExamplesTest.fiveStepPredictions["opf"][i],
+                        ExamplesTest.fiveStepPredictions["network"][i])
+
+
+  @unittest.skip
+  def testOneStepConfidencesOpfVsAlgo(self):
+    """Make sure one-step confidences are the same for OPF and Algo API."""
+    for i in range(len(ExamplesTest.oneStepConfidences["opf"])):
+      self.assertEquals(ExamplesTest.oneStepConfidences["opf"][i],
+                        ExamplesTest.oneStepConfidences["algo"][i])
 
 
-  def testOPFExample(self):
-    """Make sure the OPF example does not throw any exception"""
-    sys.path.insert(0, os.path.join(self.examplesDir, "opf"))  # Add to path
-    _runExample()
+  @unittest.skip
+  def testOneStepConfidencesOpfVsNetwork(self):
+    """Make sure one-step confidences are the same for OPF and Network API."""
+    for i in range(len(ExamplesTest.oneStepConfidences["opf"])):
+      self.assertEquals(ExamplesTest.oneStepConfidences["opf"][i],
+                        ExamplesTest.oneStepConfidences["network"][i])
 
 
-  def testNetworkAPIExample(self):
-    """Make sure the network API example does not throw any exception"""
-    sys.path.insert(0, os.path.join(self.examplesDir, "network"))  # Add to path
-    _runExample()
+  @unittest.skip
+  def testOneStepConfidencesAlgoVsNetwork(self):
+    """Make sure one-step confidences are the same for Algo and Network API."""
+    for i in range(len(ExamplesTest.oneStepConfidences["algo"])):
+      self.assertEquals(ExamplesTest.oneStepConfidences["algo"][i],
+                        ExamplesTest.oneStepConfidences["network"][i])
 
 
-  def testAlgoExample(self):
-    """Make sure the algorithm API example does not throw any exception"""
-    sys.path.insert(0, os.path.join(self.examplesDir, "algo"))  # Add to path
-    _runExample()
+  @unittest.skip
```

Comment:
Because they are currently failing. This PR focuses on adding tests to see if OPF, Network and Algo API give the same results. They don't, so I am marking these tests as as skipped for now, until we figure out why we get different results (out of the scope of this PR and I need help with someone familiar with the algos)